### ISSUE TITLE
mt48lc16m16a2 wrapper patch

### ISSUE THREAD

**Issue by nturley** [openrisc/orpsoc-cores#101]
I've been working on trying to get fusesoc to run on windows and mt48lc16m16a2 is giving me problems.

From looking at the [patch file](https://github.com/openrisc/orpsoc-cores/blob/12ca9e1eb285b141dd40e59fe5dcd517406a00de/cores/mt48lc16m16a2/patches/0005-Add-wrapper.patch#L4), it looks like it is trying to put a new file in a subdirectory 
but in the [core file](https://github.com/openrisc/orpsoc-cores/blob/12ca9e1eb285b141dd40e59fe5dcd517406a00de/cores/mt48lc16m16a2/mt48lc16m16a2.core#L8), it looks like it expects the two files to be side by side, so it can't find the wrapper.

Does the linux patch utility ignore the directory in the patch or should fusesoc be checking subdirectories for source files?

**Comment by olofk** [comment]
They are supposed to be in the same directory. I'm not sure why they end up in a new directory in Windows. I remember that it was particularly tricky to apply patches to that file since it was using windows line endings. That said, there are a few options how to solve this. Since I wrote that wrapper, we have added support to FuseSoC to just override files from the repo by putting them in the core directory. That means that we can just put the wrapper file directly into the core directory, and remove the patch that adds the file.

It has also been my intention to replace the call to 'patch' with 'git apply' to reduce the number of external tools that we have to rely upon (git is already a dependency)

For now I would propose dropping the patch and just adding the file
